src/index.html

```diff
@@ -21,7 +21,402 @@
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review-amount">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#buy-link"
+          class="button card__button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review-amount">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#buy-link"
+          class="button card__button"
+        >
+          Buy
+        </a>
+      </div>
+      <div class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="iMac image"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__review-amount">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#buy-link"
+          class="button card__button"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
The card elements are repeated multiple times with the same content. Consider using a template or a loop in your JavaScript to dynamically generate these cards if they are meant to display similar data. This will make your code more maintainable and reduce redundancy.